app/views/kitchens/index.html.haml

```diff
@@ -1,17 +1,11 @@
-%h1.text-3xl.md:text-5xl.pb-4
+%h1.text-3xl.md:text-5xl.pb-8
   Konyhák
-= button_tag "Example", class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-%table.mt-4
-  %thead
-    %tr
-      %th Floor
-      %th{:colspan => "3"}
-  %tbody
-    - @kitchens.each do |kitchen|
-      %tr
-        %td= kitchen.floor
-        %td= link_to 'Show', kitchen
-        %td= link_to 'Edit', edit_kitchen_path(kitchen)
-        %td= link_to 'Destroy', kitchen, method: :delete, data: { confirm: 'Are you sure?' }
-.my-3.text-right
-  = link_to 'New Kitchen', new_kitchen_path, class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
+
+.md:grid.md:grid-cols-2.md:gap-4.mb-8
+  - @kitchens.each do |kitchen|
```

Comment:
I think kitchens should be sorted, because now they appear in creation order and that can be finnicky to control
Something like 
`@kitchens.sort_by{|kitchen| kitchen.floor}.each (...)`
could work.

Reply:
This could be done shorter with the ruby symbol to proc operator, like this: @kitchen.sort_by(&:floor)

I would put the sort in the controller action because sorting is not the views responsibility.

It's not totally mandatory to sort he kitchens, because we will seed the database in the correct order, but it's good to explicitly specify it for some edge case.

Reply:
Okay, gonna code it in, and gonna validate kitchen's floor's uniqueness